package.json

```diff
@@ -1,27 +1,27 @@
 {
         "name": "mako",
         "description": "Storage backend for Manta",
-        "version": "1.2.0",
+        "version": "2.0.0",
         "author": "Joyent (joyent.com)",
         "license": "MPL-2.0",
         "private": true,
         "dependencies": {
                 "assert-plus": "1.0.0",
                 "bunyan": "1.8.12",
-                "carrier": "0.1.7",
-                "manta": "5.2.1",
-                "posix-getopt": "1.0.0",
+                "kthxbai": "0.4.0",
                 "restify": "4.3.4",
+                "@smaller/tap": "11.1.4-1.0.0",
```

Comment:
Let me know if you need a more modern tap version here.

Did you want to consider a separate tarball for test deps? We don't do that for any repos currently, but it *could* be done. It would be a bit more setup for nightly testing setup... but might be worth it to keep the image size down (and not need to use hacks like "@smaller/...").

Reply:
I'm fine with this version and I generally prefer that the tests are included. But if we want to do something differently, we can do that too. Removing this saves 4.2M of uncompressed space (812K compressed), but that also includes some more stuff we could kthxbai if we wanted. Like 300+K of bluebird browser stuff for example. I'm not sure it's worth doing for space savings anyway.